spec/models/job_opportunity_spec.rb

```diff
@@ -1,9 +1,56 @@
 require "rails_helper"
 
 RSpec.describe JobOpportunity, type: :model do
```

Comment:
As I see we don't have any code related to job opportunities left (besides models code). Maybe we should delete the remaining code and the tables (if that's possible)?

Reply:
Yes I agree but I wanted to do it in a separate PR because this one is already big 😭